lib/govuk_design_system_formbuilder/elements/radios/fieldset_radio_button.rb

```diff
@@ -19,8 +19,8 @@ def initialize(builder, object_name, attribute_name, value, label:, hint:, link_
           @link_errors     = has_errors? && link_errors
           @html_attributes = kwargs
 
-          if block_given?
-            @conditional_content = wrap_conditional(block)
+          if block_given? && (conditional_block_content = block.call.presence)
```

Comment:
Is there a reason for the different conditional statement here and at line 24: of the `fieldset_check_box.rb`

```
if block_given? && (conditional_block_content = block.call.presence)
```

and 

```
if (conditional_block_content = block_given? && block.call.presence)
```

I also wonder if it could it be refactored to:

```
if (conditional_block_content = block&.call&.presence)
```

but have not tried

Reply:
🤦🏽 thanks, fixed